docs/resource-sharing/os-backfill-containers.md

```diff
@@ -27,34 +27,145 @@ In order to configure the container, you will need:
 Running the Container with Docker
 ---------------------------------
 
-The Docker image is kept in [DockerHub](https://hub.docker.com/r/opensciencegrid/osgvo-docker-pilot) and requires a
-number of environment variables to be set in order to function appropriately:
+The Docker image is kept in [DockerHub](https://hub.docker.com/r/opensciencegrid/osgvo-docker-pilot).
+In order to successfully start payload jobs:
 
-1. Set the `TOKEN` environment variable to the authentication token you received from OSG.
-2. Set `GLIDEIN_Site` and `GLIDEIN_ResourceName` to match the site name and resource name you registered in topology,
+
+1. **Configure authentication:**
+   Open Science Pool (OSP) administrators can provide the token,
+   which you can then pass to the container by volume mounting it as a file under `/etc/condor/tokens-orig.d/`.
+   If you are using Docker to launch the container, this is done with the command line flag
+   `-v /path/to/token:/etc/condor/tokens-orig.d/flock.opensciencegrid.org`.
+   (Replace `/path/to/token` with the full path to the token you obtained from the OSP administrators.)
```

Comment:
You revised the phrasing here but did not drop the parens like in the other location